src/BandcampDownloader/Properties/Resources.de.resx

```diff
@@ -0,0 +1,444 @@
+Entspricht dem setzen von Nicht verändern bei jedem tag.</value>
+  </data>
+	<data name="checkBoxOneAlbumAtATime" xml:space="preserve">
+    <value>Ein album auf einmal _Herunterladen.</value>
+  </data>
+	<data name="checkBoxOneAlbumAtATime_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt wird nur ein Album auf einmal heruntergeladen.
+Wenn nicht werden alle auf einmal heruntergeladen (Empfohlen wenn eine hohe bandbreite zur verfügung steht).</value>
+  </data>
+	<data name="checkBoxRetrieveFilesSize" xml:space="preserve">
+    <value>_Dateigrößen vorkalkulieren.</value>
+  </data>
+	<data name="checkBoxRetrieveFilesSize_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt werden die Dateigrößen im vornherein abgerufen so dass eine genaue verbleibende Dauer berechnet werden kann.
+Das Herunterladen geht schneller wenn diese Option ausgeschaltet ist.</value>
+  </data>
+	<data name="checkBoxSaveCoverArtInFolder" xml:space="preserve">
+    <value>In _Ordner speichern</value>
+  </data>
+	<data name="checkBoxSaveCoverArtInFolder_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt wird das Cover in den Album Ordner heruntergeladen.</value>
+  </data>
+	<data name="checkBoxSaveCoverArtInTags" xml:space="preserve">
+    <value>In _Tags speichern</value>
+  </data>
+	<data name="checkBoxSaveCoverArtInTags_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt wird das Cover in den tracks gespeichert</value>
+  </data>
+	<data name="checkBoxVerboseLog" xml:space="preserve">
+    <value>_Ausführliches Log anzeigen</value>
+  </data>
+	<data name="checkBoxVerboseLog_ToolTip" xml:space="preserve">
+    <value>Wenn ausgewählt werden mehr Informationen ausgegeben</value>
+  </data>
+	<data name="folderBrowserDialogDescription" xml:space="preserve">
+    <value>Ordner auswählen um </value>
+  </data>
+	<data name="labelAlbumArtist" xml:space="preserve">
+    <value>Album Art_ist</value>
+  </data>
+	<data name="labelAlbumTitle" xml:space="preserve">
+    <value>Album _Name</value>
+  </data>
+	<data name="labelAllowedFileSizeDifference" xml:space="preserve">
+    <value>Erlaubte _dateigrößen unterschied in Prozent</value>
+  </data>
+	<data name="labelArtist" xml:space="preserve">
+    <value>_Artist</value>
+  </data>
+	<data name="labelButtonResetSettings" xml:space="preserve">
+    <value>Einstellungen _Zurücksetzen</value>
+  </data>
+	<data name="labelComments" xml:space="preserve">
+    <value>Komm_entare</value>
+  </data>
+	<data name="labelCoverArtInFolderMaxSize" xml:space="preserve">
+    <value>Maximale Größe (_px)</value>
+  </data>
+	<data name="labelCoverArtInTagsMaxSize" xml:space="preserve">
+    <value>Maximale Größe (p_x)</value>
+  </data>
+	<data name="labelDownloadMaxTries" xml:space="preserve">
+    <value>_Maximale Dwnloadversuche</value>
+  </data>
+	<data name="labelDownloadRetryCooldown" xml:space="preserve">
+    <value>Wiederholungs Q_artezeit</value>
```

Comment:
Did you mean "Wartezeit" like on the tooltip (l.408 and 417)?

Reply:
Oh yes!